src/sqlancer/clickhouse/ClickHouseProvider.java

```diff
@@ -143,8 +143,8 @@ public SQLConnection createDatabase(ClickHouseGlobalState globalState) throws SQ
         }
         con.close();
         con = DriverManager.getConnection(
-                String.format("jdbc:clickhouse://%s:%d/%s?socket_timeout=300000%s", host, port, databaseName,
-                        clickHouseOptions.enableAnalyzer ? "&allow_experimental_analyzer=1" : ""),
+                String.format("jdbc:clickhouse://%s:%d/%s?allow_suspicious_indices=1&socket_timeout=300000%s%s", host,
```

Comment:
why should we change `socket_timeout=300000%s` to `socket_timeout=300000%s%s`

Reply:
I think it's because of the additional argument `clickHouseOptions.enableAnalyzer ? "&allow_experimental_analyzer=1" : ""`, but not sure why the tests are currently failing.